### DISCUSSION TITLE
stream infer can support default speaker?

### DISCUSSION THREAD

**Discussion by estuday** [coqui-ai/TTS#4081] [General Q&A]
hi,
i followed the code in https://docs.coqui.ai/en/dev/models/xtts.html#streaming-manually ,
i found that it could not pass the speaker param and must pass reference.wav, can stream infer support use default speaker?

**Comment by eginhard** [comment]
Yes, you can get the latents and embedding for default speakers like this:
```python
gpt_cond_latent, speaker_embedding = model.speaker_manager.speakers[speaker_name].values()
```
